cmd/context/create.go

```diff
@@ -380,6 +380,28 @@ func (c ContextCommand) getUserContext(ctx context.Context, ns string) (*types.U
 				return nil, fmt.Errorf(oktetoErrors.ErrCorruptedOktetoContexts, config.GetOktetoContextsStorePath())
 			}
 		}
+
+		kubetoken, err := client.Kubetoken().GetKubeToken(okteto.Context().Name, okteto.Context().Namespace)
+		if err != nil {
+			if utils.LoadBoolean(model.OktetoUseStaticKubetokenEnvVar) {
+				oktetoLog.Info("You are using static kubernetes tokens because you set '%s', however the feature is not available and okteto CLI will be using a static token", model.OktetoUseStaticKubetokenEnvVar)
```

Comment:
I think this message is not right. We are saying that user will use static tokens because they are using the environment variable, but then, we say that the feature is not available and the CLI will use a static token.

Well, isn't that what we want setting that environment variable?